.circleci/config.yml

```diff
@@ -65,14 +65,14 @@ jobs:
     - checkout
     - setup_remote_docker:
         docker_layer_caching: true
+    - run: sudo apt-get update
```

Comment:
This change is already in master - if you rebase and then force push the change will come in without having to merge another PR in directly.